documentation/src/main/asciidoc/topics/con_cache_store_sql.adoc

```diff
@@ -0,0 +1,22 @@
+[id='sql-cache-store_{context}']
+= SQL cache stores
+
+SQL cache stores pre-load {brandname} caches from databases, with any dialect, through JDBC connections.
```

Comment:
Any dialect is a bit broad. The ones we support are at https://github.com/infinispan/infinispan/blob/main/persistence/jdbc-common/src/main/java/org/infinispan/persistence/jdbc/common/DatabaseType.java#L10-L26

Reply:
@wburns Good point. I looked in Javadocs but don't see that enum in 13. It seems the problem is that Javadocs are included for the org.infinispan.persistence.jdbc.common.configuration package.

It should be org.infinispan.persistence.jdbc.common.